doc/source/whatsnew/v1.4.4.rst

```diff
@@ -46,7 +46,8 @@ Bug fixes
 - Bug in :meth:`.DataFrameGroupBy.value_counts` where ``subset`` had no effect (:issue:`46383`)
 - Bug when getting values with :meth:`DataFrame.loc` with a list of keys causing an internal inconsistency that could lead to a disconnect between ``frame.at[x, y]`` vs ``frame[y].loc[x]`` (:issue:`22372`)
 - Bug in the :meth:`Series.dt.strftime` accessor return a float instead of object dtype Series for all-NaT input, which also causes a spurious deprecation warning (:issue:`45858`)
-
+- Bug in :meth:`SQLTable._get_dtype` treating sqlalchemy.dialects.oracle.NUMBER as sqlalchemy.types.Integer which results in lost precision
```

Comment:
Whatsnew notes should be more user-facing, i.e. not really discussing the internal places where a fix was made. I think you want to reword this along the lines of `Bug in pd.read_sql...for Oracle databases where NUMBER types that are actually integers would still get interpreted as floats and lose precision`

Reply:
Acknowledged and changed :)